src/languages/yaml.js

```diff
@@ -10,6 +10,10 @@ Category: common, config
 export default function(hljs) {
   var LITERALS = 'true false yes no null';
 
+  // YAML spec allows non-reserved characters in tags
+  var NON_EX_CHARS = '\\w#;/?:@&=+$,.~*\\\'()[\\]'
```

Comment:
What is "EX"?

Reply:
It was originally `NON_EXCLAMATION_CHARS` but that was too long. Maybe `NON_PREFIX_CHARS` is better?

Reply:
It wasn't too long.

Reply:
Wait, are they non prefix character or prefix characters?

Reply:
You're right, prefix is confusing (here I meant '!' as prefix). Going back to exclamation.